app/api/private-cloud/products/[licencePlate]/comments/_operations/delete.ts

```diff
@@ -1,3 +1,12 @@
-export default async function deleteOp() {
-  return { message: 'Delete comment successful' };
+import prisma from '@/core/prisma';
+
+export async function deleteOp(licencePlate: string, commentId: string) {
+  const deleteResponse = await prisma.privateCloudComment.deleteMany({
```

Comment:
`deleteMany` deletes more than one items; there is another method to delete one only.